### PR TITLE
[FOSSA] Add Code Analyzers section and Connect to Fossa UI

### PR DESCRIPTION
**Pull request by angieyeh**
Task: https://issues.newrelic.com/browse/NR-119840

We are adding a new section for Code Analyzers. This PR only contains the UI for the new section. 

When a repo is not open, we do we not show the repo name in the section header.
Screenshot 2023-06-12 at 3 54 46 PM
When a repo is open, we display the repo name next to the section header.
Screenshot 2023-06-12 at 3 57 13 PM

### CODE REVIEW

**Comment by bcanzanella** [comment]
@angieyeh I changed the base branch from develop to `feature/fossa`. that way, we don't have to worry about accidental merges into develop. You can continue to make PRs against `feature/fossa` and eventually a PR from `feature/fossa` to `develop` can be made.

File: shared/ui/Stream/CodeAnalyzers/CodeAnalyzers.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+export const CodeAnalyzers = (props: Props) => {
+	const derivedState = useSelector((state: CodeStreamState) => {
+		const { editorContext, providers } = state;
+
+		const providerInfo: { [key: string]: object | undefined } = {};
+		for (const provider of ["fossa*com"]) {
+			const name = providers[provider]?.name;
+			if (name) {
+				const p = getUserProviderInfoFromState(name, state);
```
Comment (eric-j) [inline shared/ui/Stream/CodeAnalyzers/CodeAnalyzers.tsx:24]: Make this variable a bit more descriptive.

**Review by eric-j — APPROVED** [review]
Looks great! One small comment about variable name, but feel free to merge to `feature/fossa`.

File: shared/ui/Stream/CodeAnalyzers/CodeAnalyzers.tsx (hunk 1 above)
Comment (angieyeh) [reply shared/ui/Stream/CodeAnalyzers/CodeAnalyzers.tsx:24]: good call, will make the change.

**Pull request merged**